RELEASING.md

```diff
@@ -12,29 +12,39 @@ Check [workflows/test-suite.yml](https://github.com/mongodb/terraform-provider-m
+- Include the PM as a PR reviewer
+- Contact Documentation team in Slack to review the PR
+- Example: [#1478](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1478).
+
+The PR includes the following changes:
+
+#### Generate the CHANGELOG.md 
 We use a tool called [github changelog generator](https://github.com/github-changelog-generator/github-changelog-generator) to automatically update our changelog. It provides options for downloading a CLI or using a docker image with interactive mode to update the CHANGELOG.md file locally.
 
 - Update `since_tag` and `future-release` in [.github_changelog_generator](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github_changelog_generator)
 - **There is a bug with `github_changelog_generator` ([#971](https://github.com/github-changelog-generator/github-changelog-generator/issues/971))**: Make sure to update the `future-tag` with the pre-release tag. Once you generate the changelog, update `future-tag` with the final release tag in [.github_changelog_generator](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github_changelog_generator). Then, manually update the generated changelog to remove references to the pre-release tag
 - Run the following command: 
     ```bash 
-    docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator -u mongodb -p terraform-provider-mongodbatlas -t <GH_TOKEN> --breaking-labels "breaking-change" --enhancement-label "**Enhancements**" --bugs-label "**Bug Fixes**"  --issues-label "**Closed Issues**" --pr-label "**Internal Improvements**"
+    docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator -u mongodb -p terraform-provider-mongodbatlas -t <GH_TOKEN> --breaking-labels "breaking-change" --enhancement-label "**Enhancements**" --bugs-label "**Bug Fixes**"  --issues-label "**Closed Issues**" --pr-label "**Internal Improvements**" --max-issues 1000
     ```
     To obtain your github personal access token you can use the following guide: [Authorizing a personal access token for use with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)
-- Create a JIRA ticket and open a PR against the **master** branch. Make any manual adjustments if needed taking into account date format and format parameter names and resources/data source names if they begin with `mongodbatlas`.
-- Include the PM as a PR reviewer
-- Contact Documentation team in Slack to review the PR
-- Example: [#1478](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1478). 
-- If the right most version digit is 0 then create a new doc in /website/docs/guides/X.Y.0-upgrade-guide.html
```

Comment:
have we deleted the doc about when creating a new upgrade guide?

Reply:
GH diff is a little confusing, but the change is quite the contrary. Defined a separate title for the upgrade guide to give it more visiblity: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/d15f6e96d7b4d4823a23cda2b3abd7412f8d34b7/RELEASING.md